gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,53 @@ def get_possible_database_path():
     return path
 
 
+def create_database_directory():
+    """Creates the standard GRASS GIS directory.
+
+    Creates database directory named grassdata in the standard location 
+    according to the platform.
+
+    Returns the new path as a string or None if nothing was found or created.
+    """
+    home = os.path.expanduser('~')
+
+    # Determine the standard path according to the platform
+    if sys.platform.startswith('linux') or sys.platform.startswith('darwin'):
```

Comment:
Hopefully the last change, sorry I missed this before: there are other platforms (see https://stackoverflow.com/questions/446209/possible-values-from-sys-platform) so we should probably have it more like this (and treat windows as special case):

```
if sys.platform == 'win32':
    path = os.path.join(home, "Documents", "grassdata")
else:
    path = os.path.join(home, "grassdata")
```